padglobal/padglobal.py

```diff
@@ -105,6 +113,14 @@ async def red_delete_data_for_user(self, *, requester, user_id):
         """
         return
 
+    async def register_menu(self):
+        await self.bot.wait_until_ready()
+        menulistener = self.bot.get_cog("MenuListener")
+        if menulistener is None:
+            logger.warning("MenuListener is not loaded.")
+            return
+        await menulistener.register(self)
+
```

Comment:
`get_menu_default_data` or whatever it is *currently* isn't a required function but Aradia made it one (you prob didn't merge the PR yet) so can you add that? it can just return {} if you don't need dgcog or anything. I'll also try and make the docs clearer on what is and isn't mandatory

also can you merge from upstream (misc-cogs repo) (so just run `|cog update`) in case there's anything else he made mandatory that I'm not realizing about

Reply:
I don't see it required at the moment (since menulistener still checks that get_menu_default_data is a present attribute), but I've added it

Reply:
oh maybe it wasn't merged yet